lib/cfg-lexer.h

```diff
@@ -85,7 +85,12 @@ typedef struct _CfgLexerKeyword
 
 #define CFG_KEYWORD_STOP "@!#?"
 
-
+/* structure that describes a given file path type */
```

Comment:
These kinda comments does not have any value; as does not give any extra information and does not convey information which could not be delivered via code itself; so it should not be here.

Reply:
ok